packages/xdl/src/Analytics.ts

```diff
@@ -43,10 +47,10 @@ export class AnalyticsClient {
   }
 
   public logEvent(name: string, properties: any = {}) {
-    if (this.segmentNodeInstance && _userId) {
+    if (this.segmentNodeInstance && this._userId) {
       this.ensureUserIdentified();
```

Comment:
```suggestion
```
If you are calling  setUserProperties for every instance, then this line is not necessary anymore